src/command_helpers/asset_helper.rb

```diff
@@ -26,6 +26,11 @@ def assign_asset_to_node_if_given(asset_name)
         asset_cache.save
       end
 
+      def unassign_asset_from_node_if_given(asset_name, *node_name)
```

Comment:
Same comment as above. The shouldn't be an array unless it will `unassign` multiple nodes from a particular asset.